lib/modem_antenna/Kconfig

```diff
@@ -7,7 +7,7 @@
 menuconfig MODEM_ANTENNA
 	bool "Modem antenna configuration"
 	depends on NRF_MODEM_LIB
-	default y if (BOARD_NRF9160DK_NRF9160_NS || BOARD_THINGY91_NRF9160_NS)
+	default y if SOC_SERIES_NRF91X
```

Comment:
The library is intended to be enabled by default for nRF916x DKs and Thingy:91s. `SOC_SERIES_NRF91X` is enabled also for customer boards, where depending on their design the library may not be needed. Maybe it would be better to just add `BOARD_NRF9161DK_NRF9161_NS` flagging instead of using `SOC_SERIES_NRF91X`.

Reply:
My thought was that this is most likely used by other boards as well, and it was better to use the SoC series to avoid a long list of boards in the end. But I am fine with keeping it too :)